servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/DefaultMultiAddressUrlHttpClientBuilderTest.java

```diff
@@ -222,6 +223,45 @@ void internalClientsUseDifferentExecutionContextWhenConfigured() throws Exceptio
         }
     }
 
+    @Test
+    void internalClientSchemeShouldBeCaseInsensitive() throws Exception {
+        HttpExecutionContext expectedCtx = new HttpExecutionContextBuilder()
```

Comment:
For this specific test, there is no need to customize the context. Just skip all these methods on the client builder